src/interface/sparse_pack.hpp

```diff
@@ -437,8 +340,71 @@ class SparsePack : public SparsePackBase {
     const int vidx = GetLowerBound(b, t) + t.idx;
     return pack_(dir, b, vidx)(k, j, i);
   }
+
+  template <class... VTs>
+  auto GetPtrs(const int b, const TE el, int k, int j, int i, VTs... vts) {
+    return std::make_tuple({&(*this)(b, el, vts, k, j, i)...});
```

Comment:
This functionality needs to be called inside a `par_for_outer` but before a `par_for_inner`.  I worry about calling `std::make_tuple` inside a GPU kernel.  I know that `std::vector`s are not permitted, are we certain that tuples are allowed?

Reply:
After a fix and adding pulling out pointers to the sparse pack test, it seems to at least compile on the darwin nv-devkit nodes. We'll see if it passes all of the tests on GitHub